build.sbt

```diff
@@ -4,7 +4,7 @@ val commonSettings = Seq(
   name := "mantis",
   version := "1.0-daedalus-rc1",
   scalaVersion := "2.12.1",
-  testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-l", "MinerSpec") // miner tests disabled by default
+  testOptions in Test += Tests.Argument(TestFrameworks.ScalaTest, "-l", "EthashMinerSpec") // miner tests disabled by default
```

Comment:
I remember, Christos, you had a problem a with loading the DAG on CircleCI. Was it because this test was not disabled after renaming? I wonder what's the point of keeping it if it's never run...

Reply:
Indeed that is the case. Should have changed it here as well but forgot to.

You make a good point but I think it is for discussion in another venue ;)

Reply:
(@loverdos  You can bring that up on Tuesday?)